cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -67,6 +67,7 @@ func encodeConfig(cfg *config.KubeSchedulerConfiguration) (*bytes.Buffer, error)
 	var encoder runtime.Encoder
 	switch cfg.TypeMeta.APIVersion {
 	case configv1beta2.SchemeGroupVersion.String():
+		klog.Warning("KubeSchedulerConfiguration v1beta2 is deprecated in v1.25, will remove in v1.26")
```

Comment:
encodeConfig is not called if the log verbosity level is less than 2.

We should probably do this in loadConfig instead.

Reply:
Yes, updated.